packages/app-desktop/gui/note-viewer/index.html

```diff
@@ -589,6 +589,22 @@
 			let element = event.target;
 
 			// To handle right clicks on resource icons
+			let mermaidElement = element.closest("[id^='mermaid']") || ((element.firstChild && element.firstChild.id?.startsWith('mermaid')) ? element.firstChild : null);
+			if(mermaidElement) {
+				const svgString = new XMLSerializer().serializeToString(mermaidElement);
+				ipcProxySendToHost('contextMenu', {
+					type: 'image',
+					tempResource: {
+						data: svgString,
+						mime: 'image/svg+xml',
+						filename: mermaidElement.id + '.svg',
+					}
+				});
+			}
+			else{
+				console.log('mermaid element not found',mermaidElement);
```

Comment:
Doesn't it mean that every time we right click on an element that's not a mermaid graph it will show this message? Also why print mermaidElement since it's undefined?

Reply:
Oh sorry I missed this one, forgot to remove it. Will remove other logs as well if any.